custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -74,7 +73,7 @@ def get_data_for_daily_usage(date, filters):
             'color': get_color_with_green_positive(daily_attendance_percent),
             'percent': daily_attendance_percent,
             'value': get_value(daily_yesterday, 'daily_attendance'),
-            'all': get_value(daily_yesterday, 'awcs'),
+            'all': get_value(awc_this_month_data, 'awcs'),
```

Comment:
@dhiraj-beehyv  can you put these under feature flag. `pre_release_features` variable(boolean) in its caller function should have that info.